functions/index.js

```diff
@@ -1,95 +1,147 @@
-/**
- * Import function triggers from their respective submodules:
- *
- * const {onCall} = require("firebase-functions/v2/https");
- * const {onDocumentWritten} = require("firebase-functions/v2/firestore");
- *
- * See a full list of supported triggers at https://firebase.google.com/docs/functions
- */
+/* eslint-disable spaced-comment */
+/* eslint-disable comma-spacing */
+/* eslint-disable object-curly-spacing */
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable comma-dangle */
+/* eslint-disable key-spacing */
+/* eslint-disable quotes */
+/* eslint-disable indent */
+/* eslint-disable max-len */
+
```

Comment:
Please do not disable lint check, use this command to fix lint automatically


```suggestion

npx eslint . --fix

```